RaceTrack/Tournament/Tournament.cs

```diff
@@ -0,0 +1,61 @@
+﻿using DataRace;
+
+namespace RaceTournament
+{
+    public class TournamentResult
+    {
+        private Queue<Racer> _racers;
+        public Queue<Racer> _winners = new Queue<Racer>();
+        public Action<Racer, Racer, Racer> OnRaceCompleted;
```

Comment:
Create a model which will contain 3 Racers because now it's difficult to understand what are these 3 racers